### ISSUE TITLE
Please make for forge too

### ISSUE THREAD

**Issue by nohax955** [FoundationGames/Splinecart#11]
forge versions of the mod

**Comment by Lyxairo** [comment]
+1, and a 1.20.1 version support would be awesome too!

**Comment by vinnyg0621** [comment]
> +1, and a 1.20.1 version support would be awesome too!

agreed

**Comment by Lassilos** [comment]
Yeah it would be cool but very annoying to do as almost the whole code base would have to be recoded and modified.

**Comment by blackmidiuser** [comment]
Use Sinytra connecter (I did not test it yet, do it at your own risk)
